doc/source/internals.rst

```diff
@@ -152,44 +152,110 @@ Below example shows how to define ``SubclassedSeries`` and ``SubclassedDataFrame
        def _constructor_sliced(self):
            return SubclassedSeries
 
+
+Overriding constructor properties allows subclass families to be preserved across slice and reshape operations:
+
 .. code-block:: python
 
```

Comment:
actually I am puzzled why these are code-blocks at all, they should all simply be ipython-blocks. much more simple and will show an error if something is wrong. can you change.